genshin/client/components/base.py

```diff
@@ -378,21 +378,24 @@ async def request_hoyolab(
         url = routes.TAKUMI_URL.get_url(region).join(yarl.URL(url))
 
         if region == types.Region.OVERSEAS:
-            headers = {
+            _headers = {
```

Comment:
genshin.py categorically does not use private variables anywhere other than class variables. Please refer to other `request_foo` methods.

Reply:
Fixed